### ISSUE TITLE
'unzip' command is not installed by default (Ubuntu 14.04). It's required to unzip!

### ISSUE THREAD

**Issue by mscook** [Linuxbrew/linuxbrew#389]
I'm using images build by vagrant.

Solution - consider adding **unzip** to the apt-get/yum chain.

```
==> Installing nullarbor dependency: blat
==> Downloading http://users.soe.ucsc.edu/~kent/src/blatSrc35.zip
######################################################################## 100.0%
Error: Failure while executing: unzip -qq /home/vagrant/.cache/Homebrew/blat-35.zip

sudo apt-get install unzip
...
==> Installing nullarbor dependency: blat
==> Downloading http://users.soe.ucsc.edu/~kent/src/blatSrc35.zip
Already downloaded: /home/vagrant/.cache/Homebrew/blat-35.zip
==> make MACHTYPE=darwin BINDIR=/home/vagrant/.linuxbrew/Cellar/blat/35/bin
```

**Comment by jszakmeister** [comment]
FWIW, people trying to build Neovim via Linuxbrew have been running into this problem too.

**Comment by sjackman** [comment]
I had thought that this issue was already resolved by a previous pull request. It would seem not. I'm on vacation until the new year. Please bump this thread after then. Sorry for the delay.

**Comment by jszakmeister** [comment]
> I had thought that this issue was already resolved by a previous pull request. It would seem not. I'm on vacation until the new year. Please bump this thread after then. Sorry for the delay.

No worries.  I'll try and remember to bump this after the new year.  Enjoy your vacation!

**Comment by sjackman** [comment]
This issue is resolved by 519dc5ff.
